config/Config-template.groovy

```diff
@@ -155,6 +155,9 @@ environments { development {
     com.recomdata.heartbeatLaps = 900
 } }
 
+// Maximum concurrent sessions for a user (-1: unlimited)
+// com.recomdata.maxConcurrentUserSessions = 10
```

Comment:
@forus I must admit I don't have a good feeling where it should be.
My choice was only based on similarity with "com.recomdata.sessionTimeout", "com.recomdata.guestAutoLogin", "com.recomdata.passwordstrength.pattern", ...
I'm happy to follow your suggestion. I will change it.